app/model/CandidateProgressReport.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Not sure if we do it before but I think it might be a good idea to put all report case classes in its own package. The reason is they usually agregrate information from different classes. What do you think?